src/store/ChartState.js

```diff
@@ -38,14 +39,15 @@ class ChartState {
     }
 
     saveLayout() {
+        if (!this.chartId) return;
         const layoutData = this.stxx.exportLayout(true);
         const json = JSON.stringify(layoutData);
-        CIQ.localStorageSetItem(`layout-${this.id}`, json);
+        CIQ.localStorageSetItem(`${this.chartId}-layout-${this.id}`, json);
```

Comment:
with this proposed change, we don't use `this.id` anymore

Reply:
@brucebinary  As far as i understood we have one layout for each chart so im going to remove `this.id` from here. dont think its needed since now we have an id for each chart.